### PR TITLE
Create (and then discover) storage account for registry

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

**Comment by jim-minter** [comment]
@maxamillion hopefully this gives you what you need?

Re: the FIXME, I discussed that with @pweil- .

File: parts/k8s/kubernetesmasterresources.t [hunk 1]
```diff
@@ -109,6 +109,15 @@
             "name": "Basic"
         }
     },
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "[variables('apiVersionStorage')]",
+      "name": "[concat(variables('storageAccountBaseName'), 'registry')]",
+      "location": "[variables('location')]",
+      "properties": {
+        "accountType": "[variables('vmSizesMap')[variables('masterVMSize')].storageAccountType]"
```
Comment (jim-minter) [inline parts/k8s/kubernetesmasterresources.t]: See https://github.com/kargakis/acs-engine/issues/96 as a follow-up here

**Comment by maxamillion** [comment]
lgtm 👍

**Pull request merged**